### DISCUSSION TITLE
[Feature Request] nitrite-database support

### DISCUSSION THREAD

**Discussion by ran0dom0host** [dbeaver/dbeaver#17164] [Ideas]
Hopefully development team see this and please check 
https://www.dizitart.org/nitrite-database/

Nitrite-database  is an open source nosql, server-less, embedded database.It was written in Java ,It has MongoDB like API. It supports both in-memory and single file based persistent store powered engine of h2 database,i.e. MVStore.

There are currently no console or direct access to this database, it would be helpful if dbeaver could cover this.

**Comment by LonwoLonwo** [comment]
Hello @ran0dom0host 

Usually, we use JDBC drivers for connection to databases.
Do you have a link to the nitrite JDBC driver?

**Reply by ran0dom0host** [reply]
Thx for reply, I think the database do not have a JDBC driver yet.  Might be I need to wait till it gets popular~
